internals/secrethub/formatter.go

```diff
@@ -0,0 +1,193 @@
+package secrethub
```

Comment:
Should this file be called `formatter.go` or `list_formatters.go`?

Reply:
`list_formatters.go` is indeed a better name for it.